azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/ContainerService.java

```diff
@@ -0,0 +1,263 @@
+             * Specifies the Swarm orchestration type for the container service.
+             * @return the next stage of the definition
+             */
+            WithCreate withSwarmOrchestration();
+
+            /**
+             * Specifies the DCOS orchestration type for the container service.
+             * @return the next stage of the definition
+             */
+            WithCreate withDCOSOrchestration();
+
+            /**
+             * Specifies the Kubernetes orchestration type for the container service.
+             * @return the next stage of the definition
+             */
+            WithCreate withKubernetesOrchestration();
+        }
+
+        /**
+         * The stage of the container service definition allowing to specify the master profile.
+         */
+        interface WithMasterProfile {
+            /**
+             * The properties for master agents.
+             * @param count master profile count (1, 3, 5)
+             * @param dnsLabel the dns prefix
+             * @return the next stage of the definition
+             */
+            WithLinuxProfile withMasterProfile(ContainerServiceMasterProfileCount count, String dnsLabel);
+        }
+
+        /**
+         * The stage of the container service definition allowing to specify an agent pool profile.
+         */
+        interface DefineAgentPoolProfiles {
+            /**
+             * Begins the definition of a agent pool profile to be attached to the container service.
+             *
+             * @param name the name for the agent pool profile
+             * @return the stage representing configuration for the agent pool profile
+             */
+            CSAgentPoolProfile.DefinitionStages.Blank<WithCreate> defineContainerServiceAgentPoolProfile(String name);
+        }
+
+        /**
+         * The stage of the container service definition allowing the start of defining Linux specific settings.
+         */
+        interface WithLinuxProfile {
+            /**
+             * Begins the definition to specify Linux settings.
+             * @return the stage representing configuration of Linux specific settings.
+             */
+            WithLinuxProfileRootUsername withLinuxProfile();
+        }
+
+        /**
+         * The stage of the container service definition allowing to specific the Linux root username.
+         */
+        interface WithLinuxProfileRootUsername {
+            /**
+             * Begins the definition to specify Linux root username.
+             * @param rootUserName the root username
+             * @return the next stage of the definition
+             */
+            WithLinuxProfileSshKey withRootUsername(String rootUserName);
+        }
+
+        /**
+         * The stage of the container service definition allowing to specific the Linux ssh key.
+         */
+        interface WithLinuxProfileSshKey {
+            /**
+             * Begins the definition to specify Linux ssh key.

```

Comment:
casing in JavaDocs 'SSH'
(there are more of those, so pls correct all occurrences)